Entities/Industry/CTFShops/Quarters/Quarters.as

```diff
@@ -268,6 +266,7 @@ void onAttach(CBlob@ this, CBlob@ attached, AttachmentPoint@ attachedPoint)
 	attached.AddScript("WakeOnHit.as");
 
 	string texName = default_head_path;
```

Comment:
Compile error: `default_head_path` is still used.
Could set it to `'Heads.png'`.